### PR TITLE
Early return func AfterEpochEnd

### LINKED ISSUE #613: style: all: invert code and use early returns to make for direct reading without strained context and to reduce nesting
## Summary of Bug
Early returns are more idiomatic Go code and reduce the need for heavy nesting
If we look at the code all around the repository we see code nesting for example:
* Keeper.AfterEpochEnd could invert the conditions for an early return https://github.com/quicksilver-zone/quicksilver/blob/c26b22e7cf2eebef24f70289f73552f54da807ea/x/interchainstaking/keeper/hooks.go#L31-L196

can be inverted to
```go
if epochIdentifier != epochstypes.EpochIdentifierEpoch { // not at the Epoch's end!
    return nil
}

k.Logger(ctx).Info("handling epoch end", "epoch_identifier", epochIdentifier, "epoch_number", epochNumber)
```

We should do the same fixes throughout the entire repository.

## Version

13bf586baffd07ca4a2a7f093212455fafb5df27

____

#### For Admin Use

- [x] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged/assigned

### PR DESCRIPTION
**Pull request by neitdung**
## 1. Summary
Fixes #613 

## 2.Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## 5. Checklist

- [ ] Does the Readme need to be updated?

### CODE REVIEW

**Review by ThanhNhann — COMMENTED** [review]
Can you check another func has the same style like this

**Comment by neitdung** [comment]
Ok, let me do this

**Comment by neitdung** [comment]
Hi @ThanhNhann, I've read all module in `x/` folder and found 2 more issues. Please review this again.

File: x/mint/keeper/hooks.go [hunk 1]
```diff
@@ -18,56 +18,57 @@ func (k Keeper) AfterEpochEnd(ctx sdk.Context, epochIdentifier string, epochNumb
 	params := k.GetParams(ctx)
```
Comment (ThanhNhann) [inline x/mint/keeper/hooks.go:18]: I think we should move this line to after line 32 `minter := k.GetMinter(ctx)`

**Review by ThanhNhann — APPROVED** [review]
Lgtm!

**Pull request merged**